projects/storefrontlib/src/lib/ui/pages/pages.module.ts

```diff
@@ -66,6 +58,18 @@ const pageModules = [
         canActivate: [CmsPageGuards],
         component: PageLayoutComponent,
         data: { pageLabel: 'homepage', cxPath: 'home' }
+      },
+      {
+        path: null,
+        canActivate: [CmsPageGuards],
+        component: PageLayoutComponent,
+        data: { pageLabel: 'faq', cxPath: 'help' }
+      },
+      {
+        path: null,
+        canActivate: [CmsPageGuards],
+        component: PageLayoutComponent,
+        data: { pageLabel: 'termsAndConditions', cxPath: 'termsAndConditions' }
       }
```

Comment:
If we move here the routes for FAQ and T&C, shouldn't we also put here the routes for Sale and Contact pages? (As long as we don't have generic route for Content Page with `**`)

Reply:
As we do not have any content for those pages in our backends, I thought it's pointless for the moment. Moreover, we will drop the specific route config for those content pages anyway, that's why I did no longer have them.